processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -363,6 +374,22 @@ public ColumnCapabilities getColumnCapabilities(String columnName)
     {
       return delegate.getColumnCapabilities(columnName);
     }
+
+    @Override
+    public DoubleColumnSelector makeDoubleColumnSelector(String columnName)
```

Comment:
Put this method along with other similar methods and make the implementation alike